scripts/check-pkg-for-release.js

```diff
@@ -21,13 +21,188 @@
  * The argument should be the package.json file to check.
  */
 
+const assert = require('assert');
+const fs = require('fs');
 const path = require('path');
+
+const {default: traverse} = require('babel-traverse');
+const babylon = require('babylon');
+const camelCase = require('camel-case');
+const cssom = require('CSSOM');
+const recast = require('recast');
+
 const pkg = require(path.join(process.env.PWD, process.argv[process.argv.length - 1]));
-if (pkg.version !== '0.0.0') {
-  process.exit(0);
+
+const REPO_PKG = require(path.join(process.env.PWD, 'package.json'));
+const WEBPACK_CONFIG_PATH = 'webpack.config.js';
+const WEBPACK_CONFIG = require(path.join(process.env.PWD, WEBPACK_CONFIG_PATH));
+const MASTER_PKG_PATH = 'packages/material-components-web/package.json';
+const MASTER_CSS_PATH = 'packages/material-components-web/material-components-web.scss';
+const MASTER_JS_PATH = 'packages/material-components-web/index.js';
+const MASTER_PKG = require(path.join(process.env.PWD, MASTER_PKG_PATH));
+const CSS_WHITELIST = ['base', 'auto-init', 'rtl'];
+
+main();
+
+function main() {
+  checkPublicConfigForNewComponent();
+  if (pkg.name !== MASTER_PKG.name) {
+    checkNameIsAddedToAllowedScope();
```

Comment:
checkNameIsPresentInAllowedScope

us "is present in" instead of "is added to" or "added in". Its just a little more clear